mm/kfence/core.c

```diff
@@ -632,7 +633,7 @@ size_t kfence_ksize(const void *addr)
 	 * Read locklessly -- if there is a race with __kfence_alloc(), this is
 	 * either a use-after-free or invalid access.
 	 */
-	return meta ? abs(meta->size) : 0;
+	return meta ? meta->size : 0;
```

Comment:
Should we also make the size field size_t? We cast it to size_t and sizes are generally passed as size_t in slab code and memory overhead is not a concern for the meta descriptors.

Reply:
In addition to that, the comment in kfence.h needs to be updated to no longer mention 'size < 0' and 'size > 0'.

Reply:
Done